pubspec.yaml

```diff
@@ -12,5 +12,6 @@ dependencies:
   http: ">=0.12.0 <0.13.0"
 
 dev_dependencies:
-  test: ">=1.14.0 <1.15.0"
+  test: "^1.15.7"
```

Comment:
I'd prefer using a version range like before instead of a fixed version (just use `>=1.15.0 <1.16.0`). Same goes for mockito.